sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -102,144 +101,77 @@ function getRuleActionOrUndefined(value: any): SqlAction | undefined {
   }
 }
 
-/**
- * @internal
- * @ignore
- * Represents settable options on a rule
- */
-export interface RuleOptions {
-  /**
-   * @internal
-   * @ignore
-   * Defines the filter expression that the rule evaluates. For `SqlFilter` input,
-   * the expression string is interpreted as a SQL92 expression which must
-   * evaluate to True or False. Only one between a `CorrelationFilter` or
-   * a `SqlFilter` can be defined.
-   */
-  filter?: SqlFilter | CorrelationFilter;
-
-  /**
-   * @internal
-   * @ignore
-   * The SQL like expression that can be executed on the message should the
-   * associated filter apply.
-   */
-  action?: SqlAction;
-}
-
 /**
  * @internal
  * @ignore
  * Internal representation of settable options on a rule
  */
-export interface InternalRuleOptions extends RuleOptions {
+export interface InternalRuleOptions extends Pick<RuleDescription, "filter" | "action"> {
   /**
-   * @internal
-   * @ignore
    * Name of the rule.
    */
   name?: string;
 }
 
 /**
- * @internal
- * @ignore
  * Represents all attributes of a rule entity
  */
-export interface RuleDetails {
+export interface RuleDescription {
   /**
-   * @internal
-   * @ignore
    * Name of the rule
    */
   ruleName: string;
```

Comment:
This should probably be just `name`. In the API review, we talked about how queueName in QueueDescription, topicName in TopicDescription should be just `name`. `SubscriptionDescription` is the only exception because it needs to have both topic and subscription name.

Also, after doing this change, I don't see any difference between `RuleDescription` and `InternalRuleOptions`. Can we get rid of the latter?

Reply:
Done!